### ISSUE TITLE
Bavarian Grids

### ISSUE THREAD

**Issue by Bernd-Vogelgesang** [OSGeo/PROJ-data#38]
Please excuse my ignorance on the hole topic. The hole proj thing is kind of rocket science for me.
Anyway, I saw that you kind of collect grids from here and there, so I asked myself (and now you) what could be done to get those grids for the german state of Bavaria into the collection. Under https://www.ldbv.bayern.de/vermessung/utm_umstellung/trans_geofach.html one can find downloads for reference grid files for the complete state of bavaria and also more localized sub-grids for the 8 districts.
If I could be of any help bringing them in, please be so kind to guide me.
Cheers,
Bernd

**Comment by rouault** [comment]
@Bernd-Vogelgesang 

Yes, we identified that resource in https://github.com/OSGeo/proj-datumgrid/issues/76 but one blocking point is the license of the data which includes a "Non-Derivative" restriction, incompatible of the usage done by PROJ.
If that was resolved (for example by using CC-BY-4.0 like many other grids), what would help integration in PROJ would be if those grids could be registered to EPSG, like it has been done for other German grids for other Ländern (this is usually the task of the producing agency to contact EPSG to do the registration). A few hints for that mentioned in https://github.com/OSGeo/PROJ-data/blob/master/CONTRIBUTING.md

**Comment by Bernd-Vogelgesang** [comment]
@rouault Do you know by chance if anyone already tried to contact the agency about this issue? Otherwise I would give it a try.

**Comment by rouault** [comment]
> Do you know by chance if anyone already tried to contact the agency about this issue?

I'm not sure, at least there's no such trace in the ticket, so I'd say that you can give it a try.

**Comment by kbevers** [comment]
>  so I'd say that you can give it a try.

From someone who works at a similar agency: The more requests like this we get, the more likely we are to do something about. Tell them what you need and why that matters to you. They may not react at first but given enough requests for better licensed grids they can't ignore it.

**Comment by Bernd-Vogelgesang** [comment]
Email was sent this morning,asking for a change of license, and received a confirmation that it was forwarded to the appropriate department. Let's wait and see.

**Comment by Bernd-Vogelgesang** [comment]
Got a reply today (translated by deepl):

> Thank you again for your amendment.
> 
> However, the license will remain as it is for the time being. When the ntv2 file was created, further use beyond the existing license was not considered.
> 
> As on our homepage at https://www.ldbv.bayern.de/vermessung/utm_umstellung/trans_geofach.html
> 
> described above, there were already changes in the point field of the real estate cadastre after the introduction of the UTM due to new official surveys.
> 
> Thus, more and more often the provided NTv2 files can no longer establish an exact local reference to the cadastre. For this reason, it was recommended that the geotechnical data should be converted as soon as possible after the introduction of UTM.
> 
> For this reason, the ntv2-Kanu file will probably no longer be offered as a permanent download. Please understand that we will not change the licenses for the transition period.
> 
> Translated with www.DeepL.com/Translator (free version)

As I'm technically not up-to-date: Does this reply make sense to you?

**Comment by rouault** [comment]
Yes, that's make sense. So basically the incompatible licensing will remain, and they don't think that there's much point in using those files nowadays. Looks like this issue might be closed
